configs/settings-example.yml

```diff
@@ -0,0 +1,55 @@
+# Variables for ansible provisioning
+#  https://docs.ansible.com/ansible/latest/playbooks_variables.html
+
+env: "dev" # dev | production - to enable TLS change mediawiki.protocal to 'https' below
```

Comment:
Let's rather have a separate "ssl: True` so that we can drop the whole `env` thing.

Reply:
ssl is deprecated.

Reply:
Huh?

Reply:
"SSL 2.0 was prohibited in 2011 by RFC 6176, and SSL 3.0 was deprecated in June 2015 by RFC 7568" https://en.wikipedia.org/wiki/Transport_Layer_Security